src/app/jobs_service.py

```
@@ -0,0 +1,137 @@
+
+def load_job_result(result_file: str) -> Dict[str, Any] | None:
+    """Load job result from a JSON file."""
+    jobs_dir = get_jobs_data_dir()
+    result_file = jobs_dir / result_file
+    if not result_file or not os.path.exists(result_file):
+        return None
+
+    try:
+        with open(result_file, "r", encoding="utf-8") as f:
+            return json.load(f)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)